services/github.py

```diff
@@ -3,6 +3,7 @@
 
 from shared.github import InvalidInstallationError
 from shared.github import get_github_integration_token as _get_github_integration_token
+from shared.github import is_installation_rate_limited
```

Comment:
It's not an accident. It's actually a "re-export"... but yeah, doesn't make sense to do this I guess. Unless we're gonna cache it or something similar. I just wanted to keep all the "github specific things" in a single place

Notice that in `services/bots.py` it uses this function instead of importing directly from shared. 
```
from services.github import get_github_integration_token, is_installation_rate_limited
```